src/loaders/zarr/lib/utils.ts

```diff
@@ -87,13 +86,13 @@ export async function loadMultiscales(store: ZarrArray['store'], path = '') {
   };
 }
 
-/*
- * Downsampled resolutions in zarr-based image pyramids might have different
- * chunk sizes which aren't supported by our image layers.
- *
- * This function trims the pyramid to just levels with the same tilesize.
- *
- */
-export function trimPyramid<S extends string[]>(pyramid: PixelSource<S>[]) {
-  return pyramid.filter(level => pyramid[0].tileSize === level.tileSize);
+function prevPowerOf2(x: number) {
+  return 2 ** Math.floor(Math.log2(x));
+}
+
+export function guessTileSize(arr: ZarrArray) {
+  const interleaved = isInterleaved(arr.shape);
+  const [yChunk, xChunk] = arr.chunks.slice(interleaved ? -3 : -2);
+  const size = Math.min(yChunk, xChunk);
+  return prevPowerOf2(size);
```

Comment:
For some reason I was getting WebGL errors in the console for non "factor of 2" data sizes. This method inspects the chunk shape of the zarr array, and returns the nearest "power of two" for what ever chunk is smaller (x or y).

Reply:
That's interesting since we do have those settings you found
```javascript
gl.pixelStorei(GL.UNPACK_ALIGNMENT, 1);
gl.pixelStorei(GL.PACK_ALIGNMENT, 1);
```
I changed the tile size manually to 500 and didn't have an issue and also removed the power-of-two call and didn't have any issue.  Can you provide repro steps and/or logs?  There could be something we're missing

Reply:
Unrelated, We should do this same thing for `TiffPixelSource` since this issue can arise there as well.  I'll file an issue.

Reply:
Let me take a look at this tomorrow and I'll share some errors with an example

Reply:
<img width="734" alt="image" src="https://user-images.githubusercontent.com/24403730/108509563-64acd100-728b-11eb-82b4-e5d61dad2f77.png">

Reply:
firefox: 

<img width="677" alt="image" src="https://user-images.githubusercontent.com/24403730/108509680-88701700-728b-11eb-8eda-4891458241d8.png">

Reply:
This is using:

```
export function guessTileSize(arr: ZarrArray) {
  const interleaved = isInterleaved(arr.shape);
  const [ySize, xSize] = arr.chunks.slice(interleaved ? -3 : -2);
  const size = Math.min(ySize, xSize);
  return size;
 //  return 2 ** Math.floor(Math.log2(size));
}
```